app/services/app/app.ts

```diff
@@ -86,7 +87,20 @@ export class AppService extends StatefulService<IAppState> {
     await this.obsUserPluginsService.initialize();
 
     // Initialize OBS
-    obs.NodeObs.OBS_API_initAPI('en-US', electron.remote.process.env.SLOBS_IPC_USERDATA);
+    const apiResult = obs.NodeObs.OBS_API_initAPI(
+      'en-US',
+      electron.remote.process.env.SLOBS_IPC_USERDATA
+    );
+
+    if (apiResult !== EVideoCodes.Success) {
+      const message = apiInitErrorResultToMessage(apiResult);
+      showDialog(message);
+
+      crashHandler.unregisterProcess(this.pid);
+      electron.ipcRenderer.send('shutdownComplete');
```

Comment:
Looking at the way the main process handles this IPC message, I think this should work fine.